hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -776,7 +776,14 @@ private void initThreadPools(Configuration conf) {
     executorCapacity = intOption(conf,
         EXECUTOR_CAPACITY, DEFAULT_EXECUTOR_CAPACITY, 1);
     if (this.prefetchEnabled) {
```

Comment:
can you cut the this. from here and the line below. not something we do in the hadoop code except in constructors and setters

Reply:
sounds good, done